tesseract/tesseract_planning/tesseract_process_managers/src/taskflow_generators/graph_taskflow.cpp

```diff
@@ -1,242 +0,0 @@
-/**
- * @file graph_taskflow.cpp
- * @brief Creates a directed graph taskflow
- *
- * @author Levi Armstrong
- * @date August 13. 2020
- * @version TODO
- * @bug No known bugs
- *
- * @copyright Copyright (c) 2020, Southwest Research Institute
- *
- * @par License
- * Software License Agreement (Apache License)
- * @par
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- * http://www.apache.org/licenses/LICENSE-2.0
- * @par
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-#include <tesseract_common/macros.h>
-TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
-#include <console_bridge/console.h>
-#include <taskflow/taskflow.hpp>
-TESSERACT_COMMON_IGNORE_WARNINGS_POP
-
-#include <tesseract_process_managers/taskflow_generators/graph_taskflow.h>
-
-namespace tesseract_planning
-{
-GraphTaskflow::GraphTaskflow(std::string name) : name_(std::move(name)) {}
-
-const std::string& GraphTaskflow::getName() const { return name_; }
-
-tf::Taskflow& GraphTaskflow::generateTaskflow(ProcessInput input,
```

Comment:
Why can't this be converted to the new format?

Reply:
I will add it back.